themes/default/style-guide.md

```diff
@@ -0,0 +1,49 @@
+# Style Guide
+This document defines some general styles we adhere to in the docs.
+
+## Words are important
+
+- Instead of _guys_ try _folks_, _yall_, or _everyone_
+- Instead of _crazy_ try _wild_
+- Instead of _click_ use _select_
+- Instead of _dummy_ use _placeholder_
+- Avoid using gendered language
+- Avoid using violent language (e.g., _kill_)
+- Avoid pop-culture references
+
+## Headings
+
+- You should be able to scan the headings of a document and get a good sense of the page
+- Every page should have exactly one h1
+- Never skip a heading level, e.g., if your previous heading level was a 2, the next heading can be a 2 or a 3, but not a 4 or 5
+
+## Links
+
+- Link text should be descriptive and be understandable outside of the context that it is in.
+- Avoid link text like _here_, _click here_, _see here_
+
+## Notes and Warning
+
+- Use notes to communicate important information
+- Try and limit the use of notes within a single page
+- Use info for general information, and warnings for more important information
+
+### Example
+
+```
+{{% notes type="info" %}}
+This is a info note.
+{{% /notes %}}
+
+{{% notes type="warning" %}}
+This is a warning  note.
+{{% /notes %}}
+```
+
+## Quotes
+
+- Use quotes only when you are quoting a person.
```

Comment:
I think it's okay to use blockquotes for citing content in general (e.g., from other texts), so I'd broaden this slightly:

```suggestion
- Use blockquotes only when you are quoting content from another source.
```